redis_sessions/session.py

```diff
@@ -1,9 +1,9 @@
 import redis
 
 try:
-    from django.utils.encoding import force_unicode
+    from django.utils.encoding import force_text
 except ImportError:  # Python 3.*
-    from django.utils.encoding import force_text as force_unicode
+    from django.utils.encoding import force_text as force_text
```

Comment:
this does not achieve anything; catching the import error above does not solve anything, either - 
the import should fail when asking for force_text on py2 and fall back to import force_unicode - see https://docs.djangoproject.com/en/1.11/topics/python3/#string-handling

have to figure out whether force_unicode, force_text lead to different outcomes: versions of this lib that run on py2 might not be able to read sessions that are written by py3 ones

Reply:
Strangely its not effecting prod?